instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0.gradle

```diff
@@ -15,6 +15,8 @@ muzzle {
 }
 
 testSets {
+  latestDepTest
+
```

Comment:
is this needed?

Reply:
Nope, when I got rid of `latestDepTestImplementation` this line is unnecessary. I guess I was a bit confused how the new "library" thing worked.